src/test/groovy/graphql/IssueNonNullDefaultAttribute.groovy

```diff
@@ -11,16 +11,28 @@ import static graphql.schema.idl.TypeRuntimeWiring.newTypeWiring
 // See https://github.com/facebook/graphql/pull/418
 class IssueNonNullDefaultAttribute extends Specification {
     def spec = '''
+            input Locale {
```

Comment:
Not sure if it is acceptable to extend this test or if you would prefer a new test class added.